packages/shared/src/libs/helpers/format-date/format-date.helper.ts

```diff
@@ -1,8 +1,11 @@
 import { format } from "date-fns";
 
-type DateFormat = "d MMM yyyy HH:mm";
+type DateFormat = "d MMM yyyy HH:mm" | "MMM d, yyyy";
 
-const formatDate = (date: Date, dateFormat: DateFormat): string => {
+const formatDate = (
+	date: Date,
+	dateFormat: DateFormat = "d MMM yyyy HH:mm",
```

Comment:
let's always pass date format explicitly
```suggestion
	dateFormat: DateFormat,
```